src/server/utils/getIP.js

```diff
@@ -0,0 +1,12 @@
+import { networkInterfaces } from 'os';
+
+export const getIp = () => {
+  const interfaces = networkInterfaces();
+
+  const ipAddress = Object.keys(interfaces)
+    .map((name) => interfaces[name].find((iface) => iface.family === 'IPv4'
+            && !iface.internal && iface.address !== '127.0.0.1'))
+    .filter(Boolean);
+
+  return ipAddress[0] ? ipAddress[0].address : '0.0.0.0';
```

Comment:
the `ip` package actually uses `os`, it just provides more utilities, not sure if it would be better to remove it or just keep it in case we need some of the utilities in the future